manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Amazon Wish List Total",
   "short_name": "Wish List Total",
-  "version": "1.9.5",
+  "version": "1.9.6",
```

Comment:
Can you remove this? I'll bump the version on release myself.

Reply:
No worries! Thanks for creating a much needed and useful tool.